ds-ml/weather/weather-eda.py

```
@@ -0,0 +1,275 @@
+sns.set_palette("husl")
+
+def fetch_dublin_weather_2023():
+    """Fetch Dublin weather data for year 2023"""
+    
+    dublin_lat = 53.3498
+    dublin_lon = -6.2603
+    
+    start_date = '2023-01-01'
+    end_date = '2023-12-31'
+    
+    print(f"Fetching Dublin weather data for 2023...")
```

Ruff (0.13.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)